src/components/Popover.tsx

```diff
@@ -1,25 +1,27 @@
 import { Options, Placement } from '@popperjs/core'
+import { globalFontStyles } from 'css/font'
+import useInterval from 'hooks/useInterval'
+import ms from 'ms.macro'
 import maxSize from 'popper-max-size-modifier'
-import React, { createContext, PropsWithChildren, useContext, useEffect, useMemo, useRef, useState } from 'react'
+import React, { createContext, PropsWithChildren, useCallback, useContext, useMemo, useRef, useState } from 'react'
 import { createPortal } from 'react-dom'
 import { usePopper } from 'react-popper'
 import styled from 'styled-components/macro'
 import { AnimationSpeed, Layer } from 'theme'
 
-type PopoverBoundary = { boundary?: HTMLDivElement | null; updateTrigger?: any }
+type PopoverBoundary = { boundary?: HTMLDivElement | null }
 const BoundaryContext = createContext<PopoverBoundary>({})
```

Comment:
what I meant in my last review's comment abt the provider is that we can just revert back to:
```suggestion
const BoundaryContext = createContext<HTMLDivElement | null>(null)
```
since we no longer have to worry abt updateTrigger

Reply:
gotcha